lib/solr_wrapper/instance.rb

```diff
@@ -326,34 +326,27 @@ def exec(cmd, options = {})
     private
 
     def download_url
-      @download_url ||= options.fetch(:url) { default_download_url }
+      @download_url ||= config.url
+      @download_url ||= default_download_url
     end
 
     def default_download_url
```

Comment:
I tried to avoid doing this because it has actual code in there that fetches a download url.  That seems to be some other responsibility, that is not config.